### ISSUE TITLE
Добавить ресурс.

### ISSUE THREAD

**Issue by Evidark** [zhovner/zaborona_help#427]
api.bethesda.net

Пожалуйста, добавьте этот сайт в базу. У части провайдеров не работает больше года авторизация (403) в bethesda.net, хотя сам сайт работает.
 Это отражается на играх Redfall/Fallout76 и других, где нужна авторизация в Bethesda: они не работают...

**Comment by raddyst** [comment]
Это же всё на амазоне "вертится"..

**Comment by Evidark** [comment]
Вы сможете добавить этот сайт?

**Comment by raddyst** [comment]
> добавить этот сайт?

Добавлен - попробуйте.

**Comment by toxi22** [comment]
> api.bethesda.net
> 
> Пожалуйста, добавьте этот сайт в базу. У части провайдеров не работает больше года авторизация (403) в bethesda.net, хотя сам сайт работает. Это отражается на играх Redfall/Fallout76 и других, где нужна авторизация в Bethesda: они не работают...

Если Вы видите 403 ошибку, то сайт работает, но доступ блокирует либо по айпишнику либо по логину.

**Comment by Evidark** [comment]
> > добавить этот сайт?
> 
> Добавлено - попробуйте.

Прекрасно, замечательно, волшебно, это сработало, огромное спасибо!)
Как давно меня мучала эта проблема х)

**Comment by Evidark** [comment]
> Если Вы обнаружили 403 ошибку, то сайт работает, но доступ блокируется либо по айпишнику, либо по логину. Это проблема на самом сайте.

Да, я понимаю, проблема на стороне провайдера...уже полтора года или больше, так и есть. И они вообще забили на то, чтобы исправить хоть что-то. И, похоже, всех это устраивает. Меня отшивали с этим вопросом и предлагали пользоваться ВПНом...вот я и не мог найти причин и концов...

**Comment by Evidark** [comment]
Снова не работает ресурс с недавнего времени. Такая же ошибка. Буду рад, если вы снова сможете помочь с этой проблемой. Скорее всего, они обновили параметры авторизации и теперь нужно добавить следующий сайт:

accounts.bethesda.net

**Comment by toxi22** [comment]
> Снова не работает ресурс с недавнего времени. Такая же ошибка. Буду рад, если вы снова сможете помочь с этой проблемой. Скорее всего, они обновили параметры авторизации и теперь нужно добавить следующий сайт:
> 
> accounts.bethesda.net

Спасибо за информацию. Постараюсь исправить в ближайшее время
